tests/test_barostat.py

```diff
@@ -102,14 +116,14 @@ def test_barostat_varying_pressure():
     seed = 2021
     np.random.seed(seed)
 
-    box_vol = 7.8336338769085809
-    box_diff = 0.5439182266135552
+    box_vol = 7.584418174042398
```

Comment:
To clarify, these numbers have been changed because of the change in the RNG (from CPU to GPU)? I'm surprised they're *this* different. It looks like we kept the order of integration/barostat computation the same and I imagine floating point operations to nearly identical on CPU vs GPU.